src/backend/src/prisma/seed-data/work-packages.seed.ts

```diff
@@ -3,10 +3,11 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { WBS_Element_Status } from '@prisma/client';
+import { WBS_Element_Status, Work_Package_Stage } from '@prisma/client';
 
 const dbSeedWorkPackage1: any = {
   projectId: 1,
+  stage: Work_Package_Stage.RESEARCH,
```

Comment:
these need to be in work package fields (the way this is set up is nonsensical tho lmao)

Reply:
Is there any particular reason it has to go there?

Reply:
go look at how the seed data gets used in seed.ts -- it's very weird, especially when we could be using the built in prisma create types